blueocean-core-js/src/js/components/ReplayButton.jsx

```diff
@@ -29,6 +37,16 @@ export class ReplayButton extends Component {
         };
     }
 
+    componentWillReceiveProps(nextProps) {
+        const statusChanged = isRunFailed(this.props.latestRun) !== isRunFailed(nextProps.latestRun);
+
+        if (statusChanged) {
+            this.setState({
+                replaying: false,
+            });
+        }
+    }
+
```

Comment:
This fixes a bug where the button got stuck in replaying state if going through the start -> stop -> rerun cycle more than once.